app/src/main/kotlin/com/exallium/todoapp/notedetail/NoteDetailPresenter.kt

```diff
@@ -21,14 +21,13 @@ class NoteDetailPresenter(private val view: NoteDetailView,
         lookupNoteDetail(noteId)
     }
 
-    fun lookupNoteDetail(noteId: String) {
-        compositeSubscription.add(model.getNoteById(noteId).subscribe(object : SingleSubscriber<Note>() {
+    private fun lookupNoteDetail(noteId: String) {
+        compositeSubscription.add(model.getNote(noteId).subscribe(object : SingleSubscriber<Note>() {
```

Comment:
done. works wonderfully BUT it needs `AndroidSchedulers.mainThread()` which calls for Robolectric because internally the scheduler calls `Looper.getMainLooper()`. Setting up a custom `LooperFactory` is not trivial because the Looper does not have a constructor.